src/nft/types/common/common.ts

```diff
@@ -81,7 +82,7 @@ export interface Trait {
 export interface GenieAsset {
   id?: string // This would be a random id created and assigned by front end
   address: string
-  notForSale: boolean
+  notForSale?: boolean
```

Comment:
should we just cast it from gql as false if it is not provided?

Reply:
I want to avoid as many possible incorrect instantiations as possible. I think breaking is a better use experience than showing incorrect information